test/test.repo.js

```diff
@@ -425,6 +426,25 @@ describe('Creating new Github.Repository', function() {
       });
    });
 
+   it('should merge pull requests on repo', function(done) {
+      var repo = github.getRepo('michael', 'github');
+      var expectedDocUrl = 'https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button';
+
+      repo.getPull(153, function(err, pull) {
+         repo.mergePull(pull, function(err) {
```

Comment:
I meant parallel in that any user could be running them, locally, or in travis from their forks.
should this create a random commit to a new branch and merge it (with the potential to get conflicts if other is merged in between)?